docs/configuration.md

```diff
@@ -257,13 +257,12 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
-  <td><code>spark.files.userClassPathFirst</code></td>
+  <td><code>spark.executor.userClassPathFirst</code></td>
```

Comment:
the other ones like driver.extraClassPath have sparkSubmit command line options though. This one doesn't unless I missed it.